app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -12,10 +12,11 @@
             android:id="@+id/nav_tourbook"
             android:icon="@android:drawable/ic_menu_my_calendar"
             android:title="@string/menu_tourbook"/>
+        <!-- Title is correctly set in class BaseNavigationActivity -->
         <item
             android:id="@+id/nav_about"
             android:icon="@android:drawable/ic_menu_info_details"
-            android:title="Über YACguide"/>
+            android:title="@string/menu_about"/>
```

Comment:
Is there no way to substitute your app_name variable directly here in this xml file? Then it would be available at compile time and you could remove the ugly title-setting afterwards in the activity`s constructor.

Reply:
- [ ] check for substitution in xml

Reply:
I spend a lot of time to find a solution but I couldn't find a better one.

Reply:
Ok